src/components/organisms/MessagesContainer.vue

```diff
@@ -1,12 +1,13 @@
 <template>
-  <div class="flex sm:h-vh-1/2">
+  <div class="flex md:h-screen">
     <div
       class="
         w-full
         sm:w-2/5
         md:w-1/3
         sm:max-w-sm sm:border sm:border-r-0 sm:border-gray
         flex flex-col flex-shrink-0
+        px-1
```

Comment:
The px-1 does not reflect the Figma design so much and it creates a small white margin between the inbox box and the inbox items.
![image](https://user-images.githubusercontent.com/77116011/122603902-2362dd00-d043-11eb-982a-febbdcfa9481.png)

Reply:
I know,  but I wanted to see as on mobile it was getting clipped.